src/components/Modal/Modal.jsx

```diff
@@ -45,8 +46,8 @@ function Modal({
   footerContent,
 }) {
   return (
-    <div className={ `modal__wrapper ${ !isOpen ? '--hidden' : '' }` }>
-      <div className="modal__overlay" />
+    <div className={ `modal-wrapper ${ !isOpen ? '--hidden' : '' }` } data-testid="modal">
```

Comment:
Now that I look at this, negating the condition is unnecessary and makes this harder to read, get rid of the `!`

Reply:
Also weird to give the modal `wrapper` a testid of `modal`.

Reply:
Is ok if I put !isOpen in a cosnt and add it in that place?